### DISCUSSION TITLE
Exclusion programs

### DISCUSSION THREAD

**Discussion by xultrarhino** [hiddify/hiddify-app#1146] [Ideas]
Guys, what do you think of the idea to add such innovations as in AdguardVPN? So that there would be a choice of programs-exceptions and sites on which VPN will not work.

**Comment by hoobanger** [comment]
Very relevant!

**Comment by JanisRUS** [comment]
Still relevant

**Comment by n7g44ggyzp-art** [comment]
Guys how can i ruin lnuix
